app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/MatchingIdController.scala

```diff
@@ -28,38 +28,12 @@ import uk.gov.hmrc.http.HeaderCarrier
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class MatchingIdController @Inject() (
-  authAction: AuthAction,
-  featureFlags: FeatureFlags,
-  matchingService: MatchingService,
-  mcc: MessagesControllerComponents
-)(implicit ec: ExecutionContext)
+class MatchingIdController @Inject() (authAction: AuthAction, mcc: MessagesControllerComponents)
     extends CdsController(mcc) with EnrolmentExtractor {
```

Comment:
Since this class now only redirects to UserLocationController.form, could we delete the class and change the redirect in CheckYourEmailController to go straight to UserLocationController.form instead of redirecting here?

I know it's basically the same logic, but it means one less call to auth every journey, and less classes for live services to maintain